app/livechat/client/views/app/tabbar/chatRoomHistoryItem.html

```diff
@@ -0,0 +1,26 @@
+<template name='chatRoomHistoryItem'>
+	<li class='list-chat' id='{{_id}}' aria='{{v.token}}' >
+		<div class="history-div">
+			<div class="visitor-img">
+				<img src="/avatar/{{responseBy.username}}" alt="">
+			</div>
+			<div class="visitor-history-details">
+				<div class="visitor-details">
+					<h1>{{responseBy.username}}</h1>
+						{{#if open}}
```

Comment:
This template will only display closed conversations, so this `if` condition does not make sense.
Please, remove it.

Reply:
removed